app/assets/stylesheets/accounts.scss

```diff
@@ -39,7 +39,6 @@ $_current-indicator-width: 4px;
 
   &:not(:focus):hover {
     color: $govuk-link-colour;
-    text-decoration: underline;
```

Comment:
Why do we need to remove this?

Reply:
The reason I removed this line is because the base styles for `govuk-link` already specify `text-decoration: underline` which makes this rule redundant as far as I can tell.

Reply:
Thank you 😃